geode-core/src/main/java/org/apache/geode/cache/client/Pool.java

```diff
@@ -250,7 +253,11 @@
    * other threads.
    *
    * If this pool is not using thread local connections, this method will have no effect.
+   *
+   * @deprecated Since Geode 1.10.0. Thread local connections are ignored. Will be removed in future
+   *             major release.
    */
+  @Deprecated
   void releaseThreadLocalConnection();
```

Comment:
you could also mark this one as "default" and have it do nothing as the default impl

Reply:
This one is totally doable since it doesn’t log.

Reply:
Check out this change now, like?